src/orders/controller/orders.controller.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  ParseUUIDPipe,
+  Post,
+  Put,
+} from '@nestjs/common';
+import OrdersMapper from '../mapper/orders.mapper';
+import { OrdersService } from '../service/orders.service';
+import { CustomersService } from 'src/customers/service/customers.service';
+import OrderDTO from '../dto/order.dto';
+import { ApiResponse } from '@nestjs/swagger';
+import { OrderDetailsService } from '../service/orderDetails.service';
+import OrderDetailsMapper from '../mapper/orderDetails.mapper';
+
+@Controller('orders')
+export class OrdersController {
```

Comment:
Just some small fixes to do: 
- Remove what you don't use from the constructor: e.g. `private orderDetailsService: OrderDetailsService`. Check also `ProductCategoriesController` (`private productsService: ProductsService`)
- Delete commented code `// await this.ordersService.validateOrder(orderDTO.orderDetails);`
- Recheck the logic in `createOrder` (redundant initialization)

![image](https://github.com/user-attachments/assets/286f3037-9e19-4332-aa28-4603c192be87)

Reply:
done